demo/src/nextjournal/clojure_mode/demo/sci.cljs

```diff
@@ -26,7 +25,8 @@
   true)
 
 (j/defn eval-cell [on-result ^:js {:keys [state]}]
-  (-> (str "(do " (.-doc state) " )")
```

Comment:
No need for do wrapping, sci eval does a parse-next + eval loop, returning the last evaluation.